### PR TITLE
Miscellaneous cleanup of code, models, etc.

### PR DESCRIPTION
**Pull request by kdmccormick**
@edx/masters-neem 

I figured deleting code and models would be safer to do before we have production users.

After merging this, we will deploy, then merge and deploy https://github.com/edx/registrar/pull/84

### CODE REVIEW

File: registrar/apps/enrollments/migrations/0006_remove_learner.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.20 on 2019-05-06 22:19
+from __future__ import unicode_literals
```
Comment (schenedx) [inline registrar/apps/enrollments/migrations/0006_remove_learner.py]: I would prefer you merged this file after this PR is merged and deployed first.
This way, we know this PR won't cause deployment problems.

**Review by schenedx — CHANGES_REQUESTED** [review]
All looks good except we should keep the best practice on deleting columns or tables from our DB

File: registrar/apps/enrollments/migrations/0006_remove_learner.py (hunk 1 above)
Comment (kdmccormick) [reply registrar/apps/enrollments/migrations/0006_remove_learner.py]: @schenedx Good point. I put the migration file on a separate branch.

**Review by schenedx — APPROVED** [review]
👍

**Pull request merged**